e2e/utils.go

```diff
@@ -11,10 +11,10 @@ import (
 
 	"k8s.io/klog"
 
-	_ "github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1alpha1"                             // nolint
-	_ "github.com/kubernetes-csi/external-snapshotter/pkg/client/clientset/versioned/typed/volumesnapshot/v1alpha1" // nolint
-	. "github.com/onsi/ginkgo"                                                                                      // nolint
-	. "github.com/onsi/gomega"                                                                                      // nolint
+	// _ "github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1alpha1"                             // nolint
```

Comment:
Why comment these out and not remove them? If it is useful to keep commented out, add a comment describing it.

Reply:
This is left out  when we commented out the E2E related to snapshot, this need to be commented out not  removed

Reply:
yeah.